uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar-legend.component.js

```diff
@@ -18,7 +18,15 @@ define([
     $log.debug('Component: leave-calendar-legend');
 
     var vm = this;
+
     vm.legendCollapsed = true;
+    vm.other_badges = [
+      { label: 'AM', hint: 'AM Only' },
+      { label: 'PM', hint: 'PM Only' },
+      { label: 'HH:MM', hint: 'Time', type: 'hours' },
+      { label: '', hint: 'Requested', type: 'requested' },
```

Comment:
Why the label is empty here?

Reply:
Because it is empty as per mockups. If you have a look at the screenshots you may notice that the "requested" badge is empty (does not have any label inside).